src/query/algorithms/triangles/StreamingTriangles.cpp

```diff
@@ -69,33 +70,31 @@ std::map<long, std::unordered_set<long>> StreamingTriangles::getCentralAdjacency
     GraphConfig gc;
     gc.graphID = graphID;
     gc.partitionID = partitionID;
-    gc.maxLabelSize = 43;
+    gc.maxLabelSize = std::stoi(Utils::getJasmineGraphProperty("org.jasminegraph.nativestore.max.label.size"));
```

Comment:
Can we change this value arbitrarily? Is it safe to allow to do so? Have we tested the entire range of possibilities here?

Reply:
maxLabelSize needs to be greater than or equal to 4. This condition is checked in the code when creating a NodeManager object. Therefore, it is safe to do. 
We are storing nodeID in the label.  I can't guess why he tried to change the label size.

Reply:
Ok, Lets leave as it is.